### ISSUE TITLE
CPH doesn't work in vscode

### ISSUE THREAD

**Issue by sugoruru** [agrawal-d/cph#427]
Hello, I have been using the Chrome extension "Competitive Companion" for a while now.
I have been using the Chrome extension "Competitive Companion" until recently to send input/output example data to VSCode.
However, the day before yesterday, it suddenly stopped working, even though it was working until this morning.
This seems to be the same problem as #230
Also, the "Competitive Companion" seems to be able to parse and communicate with JSON files.
Here is what I did
Reinstalled Competitive Companion
Stopped other extensions to see if they work.
Change the directory of VSCode.
All of these did not work.
However, I was able to get the following two to work
Log in to the main account with the firefox extension (CPH) and run it.
Log in to the subaccount of Chrome, install CPH, and run it.
In other words, it seems that only one account in Chrome stops working.
Please reply.

**Comment by sugoruru** [comment]
Here is the VERSION information.
Extension Version: v6.1.0
VS Code Version: 1.87.2
Browser Version: chrome- 123.0.6312.88
Operating System: Windows, 11, 64 bit

**Comment by Anmol-Hanuai** [comment]
did you get the solution?

**Comment by sugoruru** [comment]
I am able to use CPH.
I will note a few methods.
- Use firefox. This is the easiest solution.
- Reinstall chrome, reinstall VSCode, reinstall extensions, etc.
It is now working fine.
